e3/os/process.py

```diff
@@ -487,14 +487,17 @@ def poll(self):
         else:
             return None
 
-    def kill(self, recursive=True):
+    def kill(self, recursive=True, timeout=3):
         """Kill the process.
 
         :param recursive: if True, try to kill the complete process tree
         :type recursive: bool
+        :param timeout: wait timeout after sending the kill signal (when
```

Comment:
Document the unit used for the timeout? (seconds?)

Reply:
I can indeed document the unit.